packages/app/cypress/e2e/studio/studio.cy.ts

```diff
@@ -255,4 +255,25 @@ it('visits a basic html page', () => {
     // Cypress in Cypress, it redirects us the the spec page, which is not what normally
     // would happen in production.
   })
+
+  it('shows menu and submenu correctly', () => {
```

Comment:
We've already got a test that opens the sub menu, don't we?

Reply:
The focus of the test is a bit different. 

As we're relocating the UI div with popperjs, it is possible that the UI div is created correctly and can be clicked with our commands but they're not visible on the screen. 

It's the bug @AtofStryker wrote [above](https://github.com/cypress-io/cypress/pull/24286#pullrequestreview-1147926810) and this test tests not to reproduce that problem again.